sdk/src/main/java/com/horizen/account/api/rpc/request/RpcRequest.java

```diff
@@ -11,15 +18,22 @@ public class RpcRequest {
     private String method;
     private JsonNode params;
 
-    public RpcRequest() {}
+    public RpcRequest(JsonNode json) throws RpcException {
+        List<String> keys = new ArrayList<>();
+        var iterator = json.fieldNames();
+        iterator.forEachRemaining(e -> keys.add(e));
+
+        if (!keys.containsAll(List.of("jsonrpc", "id", "method", "params"))) {
```

Comment:
Throwing an exception in this point can be very useful to check the correctness of a single rpc request or a multiple batch request, but if we want to manage rpc notifications we have to leave out the check on the _id_ field.

A notification is a request object without an _id_ field. A request object that is a notification signifies the client's lack of interest in the corresponding response object, and as such no response object needs to be returned to the client.

@i-Alex I ask for your confirmation on the need to handle this case as well

Reply:
yes, we need to manage notifications case as well.
Please, add it on your PR after the rebase later.